components/board.tsx

```diff
@@ -48,24 +49,18 @@ export default class Board extends React.Component<BoardProps, BoardState> {
         }
 
         return (
-            <div>
-                <div className={"status"}>{status}</div>
-                <div className={"board-row"}>
-                    {this.renderSquare(0)}
-                    {this.renderSquare(1)}
-                    {this.renderSquare(2)}
-                </div>
-                <div className={"board-row"}>
-                    {this.renderSquare(3)}
-                    {this.renderSquare(4)}
-                    {this.renderSquare(5)}
-                </div>
-                <div className={"board-row"}>
-                    {this.renderSquare(6)}
-                    {this.renderSquare(7)}
-                    {this.renderSquare(8)}
-                </div>
-            </div>
+            <Box>
+                <div>{status}</div>
+                <Grid container spacing={1}>
+                    {
+                        this.state.squares.map((value, index) => {
+                            return (<Grid item key={index} xs={4}>
+                                {this.renderSquare(index, value)}
+                            </Grid>)
+                        })
+                    }
+                </Grid>
+            </Box>
```

Comment:
```suggestion
            <Box>
                <div>{status}</div>
                <Grid container spacing={1}>
                    {
                        this.state.squares.map((value, index) => (
                            <Grid item key={index} xs={4}>
                               <Square onClick={() => this.handleClick(i)} value={value} />
                            </Grid>)
                        ))
                    }
                </Grid>
            </Box>
```

I would also avoid using index a key it is considered an anti-pattern, https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/